Makefile

```diff
@@ -63,8 +64,9 @@ else
 	yarn dist
 endif
 
+.PHONY: $(extension_node_modules)
 $(extension_node_modules):
-	cd $(@:/node_modules=) && npm install --no-audit --no-fund
+	cd $(@:/node_modules=) && yarn import && yarn install && rm yarn.lock
```

Comment:
`yarn import / install` feels hacky, maybe we can just add `--prefer-offline` to `npm install` and use `make -jX` to run these in parallel (it should not take too many seconds).

Reply:
According to the docs there is no `--prefer-offline` for `npm install` (see https://docs.npmjs.com/cli/v6/commands/npm-install)